content/tracing/advanced_usage/_index.md

```diff
@@ -1408,12 +1408,18 @@ The purpose of this section is to explain how the correlation between traces and
 {{< tabs >}}
 {{% tab "Java" %}}
 
-The idea is to leverage the MDC ([Map Diagnostic Context][1]) to automatically add the trace and span identifier into the logs.
-The Datadog Java tracer exposes two API calls to allow printing trace and span identifiers along with log statements, `CorrelationIdentifier#getTraceId()`, and `CorrelationIdentifier#getSpanId()`.
+The simplest way to inject trace identifiers is to leverage the MDC Frameworks ([Map Diagnostic Context][1]) to automatically add the trace and span ids into the logs. Datadog MDC keys may be injected automatically with a tracer integration, or manually injected with Datadog APIs.
 
-- To inject those identifier into application logs using MDC frameworks, use the following method:
+**How to enable Log Injection***
+1. Enable injection in the [Java Tracer's configuration][3]. Note: Currently only slf4j is supported for MDC autoinjection.
+1. Alternatively, Use Datadog `CorrelationIdentifier#getTraceId()`, and `CorrelationIdentifier#getSpanId()` APIs to inject identifiers at the beginning and end of each span to log (see examples below).
+2. Configure MDC to use the injected Keys
+  * `dd.trace_id` Active trace id during the log statement (or `0` if no trace)
+  * `dd.span_id` Active trace id during the log statement (or `0` if no trace)
+  * Log format must contain a snippet of both keys. Example: `dd.trace_id=%X{dd.trace_id} dd.span_id=%X{dd.span_id}`
```

Comment:
I would remove this sentence (`Log format must....`) as it is covered in the Log documentation part and the described format is actually not the correct one to use.

Reply:
Removed!